api/schemas/flow-storage.json

```diff
@@ -28,23 +28,22 @@
         }
       }
     },
-    "segments": {
-      "type": "object",
-      "description": "Mapping of timeranges (validated by regex) to segment storage locations. Format as described by the [TimeRange](../schemas/timerange#top) type",
-      "patternProperties": {
-        "^[\\[\\(]?([0-9]+:[0-9]+)?_([0-9]+:[0-9]+)?[\\]\\)]?$": {
-          "type": "object",
-          "required": [
-            "object_id"
-          ],
-          "properties": {
-            "object_id": {
-              "description": "The object store identifier for the media object covering this timerange.",
-              "type": "string"
-            },
-            "put_url": { "$ref": "http-request.json" },
-            "put_cors_url": { "$ref": "http-request.json" }
-          }
+    "media_objects": {
+      "type": "array",
+      "description": "List of information for identifying and uploading media objects",
+      "items": {
+        "type": "object",
+        "description": "Information for a media object",
+        "required": [
+          "object_id"
```

Comment:
Note a conflict with #36 which makes `put_url` required here as well

Reply:
I've made it mandatory and also changed the description to follow on from #36 - https://github.com/bbc/tams/pull/34/commits/27fb620778f0acab84d2bc362fa5bd5c3306653b